docset/windows/failoverclusters/New-Cluster.md

```diff
@@ -39,13 +39,16 @@ Use Test-Cluster to run the validation tests.
 The tests will confirm that the hardware and settings are compatible with failover clustering.
 There are multiple types of tests, including Inventory, System Configuration, Network, Storage, and other types of tests.
 
-Note: This cmdlet cannot be run remotely without Credential Security Service Provider (CredSSP) authentication on the server computer.
+> [!Note]
+> This cmdlet cannot be run remotely without Credential Security Service Provider (CredSSP) authentication on the server computer.
```

Comment:
Should we use the NOTE format in the cmdlets articles? At least we cannot do it with OfficeDocs-SkypeForBusiness and office-docs-powershell repos because Get-Help does not render it succesfully.

Reply:
Based in this document we can:
[PowerShell-Docs style guide](https://docs.microsoft.com/PowerShell/scripting/community/contributing/powershell-style-guide#markdown-extensions-supported-by-open-publishing)

I recently re-read it and decided to use it but if it is not rendering properly we may skip it.

Reply:
@get-itips what do you think?

@velkovb let's try it.

Reply:
I would vote for not using it on cmdlet's articles.

Reply:
Actually, does this content here get synced with what we get from **Get-Help**?

It appears just as plain text with my comment based help test, which is not that bad and makes the note visible.